ui/src/App.tsx

```diff
@@ -46,11 +54,26 @@ interface Purse {
 interface AppState {
   wallet?: Wallet;
   contractInstance?: unknown;
-  brands?: Array<[string, unknown]>;
-  issuers?: Array<[string, unknown]>;
+  brands?: Map<string, unknown>;
+  issuers?: Map<string, unknown>;
   purses?: Array<Purse>;
 }
 
+interface Amount {
+  brandName: string;
+  value: bigint;
+}
+
+interface FormState {
+  recipient?: string;
+  give?: object;
+  want?: object;
+}
+
+const stringify = (arg) => {
```

Comment:
I know this is just to get things working but maybe a TODO here to display amounts better would be good. Looks like it's intended for displaying non-NAT values? https://github.com/Agoric/wallet-app/blob/main/wallet/src/components/PurseValue.tsx is a good reference, though I'd prefer not to use material UI as it does.